sw/device/silicon_creator/lib/cert/asn1.h

```diff
@@ -98,7 +98,7 @@ typedef struct asn1_bitstring {
   // How many bits have been added to the current byte (between 0 and 7).
   size_t used_bits;
   // Current value of the last byte of the string.
-  size_t current_byte;
+  uint8_t current_byte;
```

Comment:
good catch, not sure why this was a `size_t` in the fist place!

Reply:
luckily the compiler caught it when I tried to compile for the device :)